src/lib.rs

```diff
@@ -81,6 +81,7 @@ extern crate slog_mozlog_json;
 extern crate slog_scope;
 extern crate slog_term;
 extern crate socketlabs;
+extern crate tokio;
```

Comment:
Does it need to be added to the lib too? Looks like only bin/queues references it?

Reply:
yes @brizental !

Reply:
Done!